avocado/utils/data_structures.py

```diff
@@ -24,7 +24,8 @@
 
 import sys
 import math
-from itertools import izip
+
+from six.moves import zip
```

Comment:
Not sure we should use this compatibility library when we can make it compatible by ourselves...
```python
try:
    # python2
    from itertools import izip as zip
except ImportError:
    # python3
    pass
```
Let's see the next commits... maybe the `six` is more useful somewhere else.